### DISCUSSION TITLE
ClassIsland BUG反映

### DISCUSSION THREAD

**Discussion by huonanwholovecomputer** [ClassIsland/ClassIsland#353] [Q&A]
问题：课前60秒提醒功能在特定情况下无法触发：

  - 在每个时间线片段中，第一节课无法触发课前提醒功能

  - 详细描述：我的时间线有三个部分：
    - 6:50-11:50为早上的部分
    - 14:10-17:25为下午的部分
    - 19:00-22:10为晚上的部分
    - 出现的问题是，6:49并没有触发课程提醒；14:09并没有触发课程提醒；18:59并没有触发课程提醒，其他的时间点正常触发课程提醒

**Comment by DryIce-cc** [comment]
给下版本号，这个问题好像修过了

**Reply by huonanwholovecomputer** [reply]
是最新的`v1.5.0.1`（`v1.5.0.0`也存在这个问题）

- 不会是我的配置的问题吧，需要测试的话，我可以通过电子邮件将所有的配置打包发送到你的邮箱
